README.md

```diff
@@ -46,6 +51,43 @@ pip to install directly from the master branch on github:
 pip install git+https://github.com/keras-team/keras-cv.git tensorflow --upgrade
 ```
 
+## Using KerasCV with Keras Core
+
+As of version 0.6.0, KerasCV supports multiple backends with Keras Core out of 
+the box. There are two ways to configure KerasCV to run with multi-backend 
+support:
+
+1. Via the `KERAS_BACKEND` environment variable. If set, then KerasCV will be 
+using Keras Core with the backend specified (e.g., `KERAS_BACKEND=jax`).
+2. Via the `.keras/keras.json` and `.keras/keras_cv.json` config files (which 
+are automatically created the first time you import KerasCV):
+- Set your backend of choice in `.keras/keras.json`; e.g., `"backend": "jax"`. 
+- Set `"multi_backend": True` in `.keras/keras_cv.json`.
+
+Once that configuration step is done, you can just import KerasCV and start 
+using it on top of your backend of choice:
+
+```python
+import keras_cv
+import keras_core as keras
+
+filepath = keras.utils.get_file(origin="https://i.imgur.com/gCNcJJI.jpg")
+image = np.array(keras.utils.load_img(filepath))
+image_resized = ops.image.resize(image, (640, 640))[None, ...]
+
+model = keras_cv.models.YOLOV8Detector.from_preset(
+    "yolo_v8_m_pascalvoc",
+    bounding_box_format="xywh",
+)
+predictions = model.predict(image_resized)
+```
+
+Until Keras Core is officially released as the new Keras, KerasCV will use 
+`tf.keras` as the default backend. To restore this default behavior, simply 
+unset `KERAS_BACKEND` and ensure that  `"multi_backend": False` or is unset in 
```

Comment:
unset `KERAS_BACKEND` -> unset the `KERAS_BACKEND` environment variable

Reply:
Maybe `unset KERAS_BACKEND`?